sklearn/linear_model/passive_aggressive.py

```diff
@@ -143,13 +134,13 @@ class PassiveAggressiveClassifier(BaseSGDClassifier):
     >>> clf.fit(X, y)
     PassiveAggressiveClassifier(C=1.0, average=False, class_weight=None,
                   early_stopping=False, fit_intercept=True, loss='hinge',
-                  max_iter=1000, n_iter=None, n_iter_no_change=5, n_jobs=None,
-                  random_state=0, shuffle=True, tol=None,
+                  max_iter=1000, n_iter_no_change=5, n_jobs=None,
+                  random_state=0, shuffle=True, tol=0.001,
                   validation_fraction=0.1, verbose=0, warm_start=False)
     >>> print(clf.coef_)
-    [[0.29509834 0.33711843 0.56127352 0.60105546]]
+    [[-0.6543424   1.54603022  1.35361642  0.22199435]]
```

Comment:
This is a really bad sign! I think we might have messed this up somehow.
I thought setting max_iter here meant we automatically set ``tol`` to 0.001 but apparently not?